### PR TITLE
add NextScan

### PR DESCRIPTION
**Pull request by MikeZeDev**

### CODE REVIEW

File: web/src/engine/websites/NextScan.ts [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+                href: string
+            }[]
+        }[]
+    }
+}
+
+@Common.MangaCSS(/^{origin}\/\d+\/\d+\/[^/]+\.html$/, 'article header h1')
+@Common.PagesSinglePageJS(pagesScript, 500)
+@Common.ImageAjax()
+export default class extends DecoratableMangaScraper {
```
Comment (ronny1982) [inline web/src/engine/websites/NextScan.ts:52]: Can't `Common.ChaptersSinglePageJS` decorator be used?

File: web/src/engine/websites/NextScan_e2e.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+const config = {
+    plugin: {
+        id: 'nextscan',
+        title: 'NextScan'
+    },
+    container: {
+        url: 'https://www.nextscanid.my.id/2023/10/a-story-about-dragon-and-rising-of.html',
+        id: '/2023/10/a-story-about-dragon-and-rising-of.html',
+        title: 'A Story About a Dragon and the Rising of an Adventurer ~ A Healer Who Was Seen as Useless and Was Kicked Out From an S Rank Party, Goes off to Revive the Strongest Dragon in an Abandoned Area'
```
Comment (ronny1982) [inline web/src/engine/websites/NextScan_e2e.ts:11]: Maybe use test case with shorter title

**Review by ronny1982 — APPROVED** [review]

File: web/src/engine/websites/NextScan_e2e.ts (hunk 2 above)
Comment (MikeZeDev) [reply web/src/engine/websites/NextScan_e2e.ts:11]: I used a longer title on purpose :D but okay.

File: web/src/engine/websites/NextScan.ts (hunk 1 above)
Comment (MikeZeDev) [reply web/src/engine/websites/NextScan.ts:52]: True, what was i thinking?

**Pull request merged**